src/LAMMPS.jl

```diff
@@ -139,13 +139,18 @@ mutable struct LMP
                 handle = API.lammps_open_no_mpi(length(args), args, C_NULL)
             end
         end
-
+        check(handle)
         this = new(handle)
```

Comment:
```suggestion
        this = new(handle)
        check(this)
```

Reply:
doing this to allow a NULL pointer to be passed to check.